src/vs/workbench/common/editor/editorStacksModel.ts

```diff
@@ -703,6 +703,7 @@ export class EditorStacksModel implements IEditorStacksModel {
 	private _onEditorDirty: Emitter<EditorIdentifier>;
 	private _onEditorLabelChange: Emitter<EditorIdentifier>;
 	private _onEditorOpened: Emitter<EditorIdentifier>;
+	private _onWillCloseEditor: Emitter<EditorIdentifier>;
 	private _onEditorClosed: Emitter<GroupEvent>;
 	private _onModelChanged: Emitter<IStacksModelChangeEvent>;
```

Comment:
Why does `onWillCloseEditor` emit a different event than `onEditorClosed`?
Intuitivly they should emit the same and it makes sense to me that is the `EditorIdentififer`